habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```diff
@@ -91,13 +99,18 @@ class PyenvAndPoetrySetup {
      * @param log                            the logger to use for output
      */
     public PyenvAndPoetrySetup(String pythonVersion, boolean usePyenv, File patchInstallScript,
-                               File baseDir, boolean rewriteLocalPathDepsInArchives, Log log) {
+                               File baseDir, boolean rewriteLocalPathDepsInArchives,
+                               String username, String password, String pypiRepoId, String pypiRepoUrl, Log log) {
         this.pythonVersion = pythonVersion;
         this.usePyenv = usePyenv;
         this.patchInstallScript = patchInstallScript;
         this.baseDir = baseDir;
         this.rewriteLocalPathDepsInArchives = rewriteLocalPathDepsInArchives;
         this.log = log;
+        this.username = username;
+        this.password = password;
+        this.pypiRepoId = pypiRepoId;
+        this.pypiRepoUrl = pypiRepoUrl;
```

Comment:
A: If we're only using this for a log statement, I'd drop it completely.

Reply:
Done